app/experimenter/normandy/tasks.py

```diff
@@ -108,20 +110,43 @@ def update_launched_experiments():
     metrics.incr("update_launched_experiments.completed")
 
 
+def is_qaOnly(recipe_data):
```

Comment:
![wx4a325srgz51](https://user-images.githubusercontent.com/119884/99833122-9e0c5e00-2b2f-11eb-887e-b6d50f17444d.jpg)

Reply:
😂 was trying to be consistent with the normandy recipe value